tests/test_dom0_systemd.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python3
+import subprocess
+import unittest
+
+# .spec file shows which units are shipped with rpm
+SYSTEM_UNITS = ["securedrop-logind-override-disable.service"]
+
+USER_UNITS = [
+    "securedrop-user-xfce-icon-size.service",
+    "securedrop-user-xfce-settings.service",
+    "sdw-notify.timer",
+]
+
+# list all system unit files (append "--user" for user files)
+# that match our naming convention
+SYSTEMD_LIST_UNITS = ["systemctl", "--legend=false", "list-unit-files", "--all", "*securedrop*"]
+
+# from systemctl status format
+SYSTEMD_EXIT_SUCCESS = "status=0/SUCCESS"
+
+
+class Dom0_Systemd(unittest.TestCase):
```

Comment:
I would personally suggest not wrapping this into a class unless there's a strong reason to do so, and just do pytest style `test_*` functions with helpers as normal functions. To make it easier whenever we get around to https://github.com/freedomofpress/securedrop-workstation/issues/998